src/components/fiter_box/constants.js

```diff
@@ -0,0 +1,11 @@
+import { t } from '../../translations';
+
+export const access_types = [[1, 'PUBLIC', t.public], [2, 'PRIVATE', t.private]];
```

Comment:
See comment below about plug_types. What if we have another access_type within API results?

Reply:
@Piiit  How can I get them from the new API?

Reply:
@fedriz Similar query as shown for `plug_types`...